tests/integration/components/o-s-s/search-field-test.ts

```diff
@@ -0,0 +1,66 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, find, click, typeIn } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import { setupIntl } from 'ember-intl/test-support';
+
+module('Integration | Component | o-s-s/search-field', function (hooks) {
+  setupRenderingTest(hooks);
+  setupIntl(hooks);
+
+  test('Search Field component is rendered with a placeholder', async function (assert) {
+    this.set('placeholder', 'This is a placeholder');
+    this.set('value', '');
+    this.set('onChange', (value: string) => {
+      this.value = value;
+    });
+
+    await render(
+      hbs`<OSS::SearchField @value={{this.value}} @placeholder={{this.placeholder}} @onChange={{this.onChange}} />`
+    );
+
+    assert.strictEqual(
+      find('input')?.getAttribute('placeholder'),
+      'This is a placeholder',
+      'The placeholder is correctly set.'
+    );
+  });
+
+  test('Input value is reseted when the clear button is clicked', async function (assert) {
+    this.set('value', 'Test');
+    this.set('onChange', (value: string) => {
+      this.value = value;
+    });
+
+    await render(hbs`<OSS::SearchField @value={{this.value}} @onChange={{this.onChange}} />`);
+
+    assert.ok(find('[role="button"]'), 'The clear button is shown.');
+
+    await click('[role="button"]');
+
+    assert.strictEqual(this.value, '', 'Input value is reseted when the clear button is clicked');
+  });
+
+  test('onChange function is triggered on input change', async function (assert) {
+    this.set('value', '');
+    this.set('onChange', (value: string) => {
+      this.value = value;
+    });
+
+    await render(hbs`<OSS::SearchField @value={{this.value}} @onChange={{this.onChange}} />`);
+    await typeIn('input', 'Test');
```

Comment:
for performance purpose, you can pass `{ delay: 0 }` as a 3rd parameter here.

This is because by default, the `typeIn` function adds some delay ([default: 50ms](https://github.com/emberjs/ember-test-helpers/blob/master/API.md#parameters-11)) between each character, leading to slower tests